components/edit-collective/sections/HostAdvancedSettings.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useMutation } from '@apollo/client';
+import { useFormik } from 'formik';
+import { cloneDeep, get, pick, set } from 'lodash';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { getErrorFromGraphqlException } from '../../../lib/errors';
+
+import Container from '../../Container';
+import HostPayouts2FARollingLimitFAQ from '../../faqs/HostPayouts2FARollingLimitFAQ';
+import { Flex } from '../../Grid';
+import MessageBox from '../../MessageBox';
+import StyledButton from '../../StyledButton';
+import StyledCheckbox from '../../StyledCheckbox';
+import StyledInput from '../../StyledInput';
+import StyledInputField from '../../StyledInputField';
+import { H3, P } from '../../Text';
+import { editCollectiveSettingsMutation } from '../mutations';
+
+import imgPreviewThread from '../../../public/static/images/conversations/conversations-list-preview.png';
+
+const messages = defineMessages({
+  'rollingLimit.label': {
+    id: 'editCollective.rollingLimit.label',
+    defaultMessage: 'Rolling payout limit',
+  },
+  'rollingLimit.placeholder': {
+    id: 'editCollective.rollingLimit.placeholder',
+    defaultMessage:
+      'For example: what type of contributors (like casinos) you do not want donations from, or under what circumstances you might allow certain donations, etc.',
+  },
+  'rollingLimit.enable': {
+    id: 'editCollective.rollingLimit.enable',
+    defaultMessage: 'Enable 2FA for payouts',
+  },
+});
+
+const ScreenshotPreview = styled.div`
+  display: block;
+  margin: 8px;
+  padding: 8px;
+  height: 240px;
+  border: 1px solid #ececec;
+  border-radius: 16px;
+
+  img {
+    max-height: 100%;
+  }
+`;
+
+const HostAdvancedSettings = ({ collective }) => {
+  const { formatMessage } = useIntl();
+  const [setSettings, { loading, error }] = useMutation(editCollectiveSettingsMutation);
```

Comment:
Should we not use `editAccountSetting` for this can of things?

Reply:
I did originally use it, but then there was an issue with getting the Apollo cache to update with the new Collective settings (which affects some of the sections rendered on the page) because the query being passed down from the edit collective page is v1. Is there a good way to fix this?

Reply:
It's always a bit tricky and we'll often face this issue when migrating to V2. The first part is to properly identify which queries / pages are not updated properly. Then either:
- We consider that the page(s) concerned by this will be migrated all at once to V2, then it's fine to use V1
- Or we use update the cache manually: https://www.apollographql.com/docs/angular/features/cache-updates/#updating-after-a-mutation
  - `update` is the best option (no additional query), but more complex
  - `refetchQueries` is a no brainer